kiss_slam/kiss_slam_pybind/occupancy_mapper/occupancy_mapper.cpp

```diff
@@ -41,21 +41,18 @@ inline float LogOddsOccupied(const float probability) {
 }  // namespace
 
 namespace occupancy_mapper {
-OccupancyMapper::OccupancyMapper(const float resolution, const float max_range)
-    : max_range_(max_range), map_(resolution), accessor_(map_.createAccessor()) {}
+OccupancyMapper::OccupancyMapper(const float resolution)
+    : map_(resolution), accessor_(map_.createAccessor()) {}
 
 void OccupancyMapper::IntegrateFrame(const Vector3fVector &pointcloud,
                                      const Eigen::Matrix4f &pose) {
     const Eigen::Matrix3f &R = pose.block<3, 3>(0, 0);
     const Eigen::Vector3f &t = pose.block<3, 1>(0, 3);
     const auto start_coord = map_.posToCoord(t);
     std::for_each(pointcloud.cbegin(), pointcloud.cend(), [&](const Eigen::Vector3f &point) {
-        const auto point_range = point.norm();
```

Comment:
I removed this `if` block from the occupancy mapper because the preprocessing pipeline (from KISS-ICP), which we use to deskew the scans post-optimization, also performs min_range and max_range filtering on the input scans. So the `max_range_` member variable is no longer necessary in this Class.